src/ja/ja.json

```diff
@@ -17,136 +17,163 @@
   },
   "data": {
     "ゞヾ": {
+      "capital": false,
       "mapping": {
         "base": "ゝヽ"
       }
     },
     "ヴ": {
+      "capital": false,
       "mapping": {
         "base": "ウ"
       }
     },
     "がガ": {
+      "capital": false,
       "mapping": {
         "base": "かカ"
       }
     },
     "ぎギ": {
+      "capital": false,
       "mapping": {
         "base": "きキ"
       }
     },
     "ぐグ": {
+      "capital": false,
       "mapping": {
         "base": "くク"
       }
     },
     "げゲ": {
+      "capital": false,
       "mapping": {
         "base": "けケ"
       }
     },
     "ごゴ": {
+      "capital": false,
       "mapping": {
         "base": "こコ"
       }
     },
     "ざザ": {
+      "capital": false,
       "mapping": {
         "base": "さサ"
       }
     },
     "じジ": {
+      "capital": false,
       "mapping": {
         "base": "しシ"
       }
     },
     "ずズ": {
+      "capital": false,
       "mapping": {
         "base": "すス"
       }
     },
     "ぜゼ": {
+      "capital": false,
       "mapping": {
         "base": "せセ"
       }
     },
     "ぞゾ": {
+      "capital": false,
       "mapping": {
         "base": "そソ"
       }
     },
     "だダ": {
+      "capital": false,
       "mapping": {
         "base": "たタ"
       }
     },
     "ぢヂ": {
+      "capital": false,
       "mapping": {
         "base": "ちチ"
       }
     },
     "づヅ": {
+      "capital": false,
       "mapping": {
         "base": "つツ"
       }
     },
     "でデ": {
+      "capital": false,
       "mapping": {
         "base": "てテ"
       }
     },
     "どド": {
+      "capital": false,
       "mapping": {
         "base": "とト"
       }
     },
     "ばバ": {
+      "capital": false,
       "mapping": {
         "base": "はハ"
       }
     },
     "ぱパ": {
```

Comment:
I know this isn't part of this PR, but is it correct that two characters are "one" diacritic?

Reply:
Oh, sorry for not responding earlier. I guess I didn't notice that Japanese was added into the mix, I was trying to target only the Latin characters with the extract method; but since it's there we could either add an exception to the extract script or leave it and wait until someone validates it.